FirebaseVertexAI/Sources/VertexAI.swift

```diff
@@ -20,70 +20,81 @@ import Foundation
 @_implementationOnly import FirebaseCoreExtension
 
 @available(iOS 15.0, macOS 11.0, macCatalyst 15.0, *)
-@objc(FIRVertexAI)
```

Comment:
Does it need to be `objc` to be initialized by the Component system?

Reply:
I _think_ so. I was seeing `Cannot convert value of type '(any VertexAIProvider).Type' to expected argument type 'Protocol'` when I dropped the `@objc`s.

Reply:
But it is dropped here?